src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -617,22 +614,22 @@ def _handle_group_actions(
         # Handle interaction actions
         for action in action_list:
             try:
-                if action.name == "status" or (
-                    use_block_kit
-                    and action.name
-                    in (
-                        "ignored:forever",
-                        "ignored:until_escalating",
-                        "unresolved:ongoing",
-                    )  # TODO: delete the first two names when block kit is GA
+                if action.name in (
+                    "status",
+                    "ignored:forever",
+                    "ignored:until_escalating",
+                    "unresolved:ongoing",
                 ):
+                    # check to see which action names are in use
+                    logger.info("slack.action.name", extra={"name": action.name})
+
                     self.on_status(request, identity_user, group, action)
                 elif action.name == "assign":
```

Comment:
might be out of scope for this PR, but i believe we should be able to get rid of this too since assignment uses the options load endpoint? i think there's actually some other things we can get rid of for assign too (the `_assign_button()` method can probably be condensed since it's overwritten as an external select anyway)

Reply:
i'll add a comment, we're looking into overhauling this function soon